app/views/partials/_home_banner.html.erb

```diff
@@ -6,7 +6,7 @@
           <h3>When homeworks are pleasure</h3>
         </div>
           <div class="centered-btn">
-          <%= link_to 'REGISTER AS A STUDENT', new_user_registration_path(teacher: false), class: 'btn btn-secondar, customized-button-std' %>
+          <%= link_to 'REGISTER AS A STUDENT', new_user_registration_path(teacher: false), class: 'btn btn-outline-dark' %>
```

Comment:
ok well, in here I do the opposite comment. We WANT them to press that button is the most important thing the user can do, so we should use btn-dark. I'd use  btn-dark for students and btn-light for teachers